### PR TITLE
make pycryptodome optional

### PR DESCRIPTION
**Pull request by nate-moo**
I can't install pycryptodome, and apparently only one provider uses it, so I propose to make it optional

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,10 +29,11 @@
         'cfscrape>=2.0.5',
         'requests-cache>=0.4.13',
         'tabulate>=0.8.3',
-        'pycryptodome>=3.8.2',
+
     ],
     extras_require={
         'selescrape': ['selenium'],
+        'pycryptodome>=3.8.2',
```
Comment (AbdullahM0hamed) [inline setup.py:36]: You should do `'key': 'value'` as it is in a dict, and will cause errors. I recommend something like: `'crypto': ['pycryptodome>=3.8.2']`

**Comment by AbdullahM0hamed** [comment]
Also, for my terrible joke:

`415: Unsupported PR Type`

**Comment by nate-moo** [comment]
> Also, for my terrible joke:
> 
> `415: Unsupported PR Type`

no

**Comment by nate-moo** [comment]
There I fixed the dictionairy

**Comment by nate-moo** [comment]
@AbdullahM0hamed I fixed the weird spacing issue

**Comment by nate-moo** [comment]
exits with this if pycryptodome isn't found
image

**Comment by vn-ki** [comment]
Build is failing. You'll probably need to update the travis config.

**Comment by nate-moo** [comment]
@vn-ki I fixed the failing build, i had to install pycryptodome manually in the travis config

**Comment by nate-moo** [comment]
no longer needed

**Pull request closed without merging**